autoload/leaderf/python/leaderf/instance.py

```diff
@@ -349,6 +350,14 @@ def _actualLength(self, buffer):
         return num
 
     def setBuffer(self, content):
+        if lfEval("g:Lf_IgnoreBufferName") == '1':
+            buffer_name = os.path.normpath(lfDecode(self._cur_buffer_name))
```

Comment:
On Windows, there may be a bug here. The path seperator of a normalized path is \, the path seperator in the content may be /.